buildSrc/src/main/java/org/elasticsearch/gradle/checkstyle/MissingJavadocTypeCheck.java

```diff
@@ -0,0 +1,162 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Can we use a normal block comment here? I believe our [notice file generation](https://github.com/elastic/elasticsearch/blob/a92a647b9f17d1bddf5c707490a19482c273eda3/buildSrc/src/main/groovy/org/elasticsearch/gradle/NoticeTask.groovy#L113) relies on that comment style.